_posts/2017-08-22-visdat.md

```diff
@@ -0,0 +1,155 @@
+
+*  I felt like the code was a little sloppy, and that it could be better.
+*  I wanted to know whether others found it useful.
+
+What I needed was someone to sit down and read over it, and tell me what they thought. And hey, a publication out of this would certainly be great.
+
+Too much to ask, perhaps? No. Turns out, not at all. This is what the rOpenSci [onboarding process](https://github.com/ropensci/onboarding) provides.
+
+### rOpenSci onboarding basics
+
+Onboarding a package onto rOpenSci is an open peer review of an R package. If successful, the package is migrated to rOpenSci, with the option of putting it through an accelerated publication with [JOSS](http://joss.theoj.org/).
+
+**What's in it for the author?**
+
+* Feedback on your package
+* Support from rOpenSci members
+* Maintain ownership of your package
+* Publicity from it being under rOpenSci
+* Contribute something to rOpenSci
+* Potentially a publication
+
+**What can rOpenSci do that CRAN cannot?**
+
+The rOpenSci onboarding process provides a stamp of quality on a package that you do not necessarily get when a package is on CRAN [^3]. Here's what rOpenSci does that CRAN cannot:
+
+- Assess documentation readability / usability
+- Provide a code review to find weak points / points of improvement
+- Determine whether a package is overlapping with another.
+
+[^3]: CRAN is an essential part of what makes the r-project successful and certainly without CRAN R simply would not be the language that it is today. The tasks provided by the rOpenSci onboarding require human hours, and there just isn't enough spare time and energy amongst CRAN managers.
+
+So I submitted visdat to the onboarding process. For me, I did this for three reasons.
+
+1. So visdat could become a better package
+2. Pending acceptance, I would get a publication in JOSS
+3. I get to contribute back to rOpenSci
+
+Submitting the package was actually quite easy - you go to [submit an issue](https://github.com/ropensci/onboarding/issues/new) on the onboarding page on GitHub, and it provides a magical template for you to fill out [^1], with no submission gotchas - this could be the future [^2]. Within 2 days of submitting the issue, I had a response from the editor, [Noam Ross](https://github.com/noamross), and two reviewers assigned, [Mara Averick](https://github.com/batpigandme), and [Sean Hughes](https://github.com/seaaan).
+
+[^1]: Never used GitHub? Don't worry, creating an account is easy, and the template is all there for you. You provide very straightforward information, and it's all there at once.
+
+[^2]: With some journals, the submission process means you aren't always clear what information you need ahead of time. Gotchas include things like "what is the residential address of every co-author", or getting everyone to sign a copyright notice.
+
+I submitted **visdat** and waited, somewhat apprehensively. _What would the reviewers think?_
+
+> **insert/link to Mara's post here?**
```

Comment:
Use this text right after _What would the reviewers think?_ as a way to link to Mara's post: 
In fact, Mara Averick wrote a post: ["So you (don't) think you can review a package"](link) about her experience evaluating `visdat` as a first-time reviewer.